internal/store/node.go

```diff
@@ -530,3 +532,12 @@ func createNodeListWatch(kubeClient clientset.Interface, _ string, _ string) cac
 		},
 	}
 }
+
+const maxValue = math.MaxInt64 / 1000
+
+func convertFloat64(q *resource.Quantity) float64 {
```

Comment:
I would call the function convertValueToFloat64 because it's not a generic float64 function